v2/api/documentdb/v1api20231115/structure.txt

```diff
@@ -0,0 +1,1781 @@
+│   │   │   │   ├── MigrationState: *Object (3 properties)
+│   │   │   │   │   ├── StartTime: *string
+│   │   │   │   │   ├── Status: *Enum (4 values)
+│   │   │   │   │   │   ├── "Completed"
+│   │   │   │   │   │   ├── "Failed"
+│   │   │   │   │   │   ├── "InProgress"
+│   │   │   │   │   │   └── "Invalid"
+│   │   │   │   │   └── TargetType: *Enum (2 values)
+│   │   │   │   │       ├── "Continuous"
+│   │   │   │   │       └── "Periodic"
+│   │   │   │   └── Type: *Enum (1 value)
+│   │   │   │       └── "Continuous"
+│   │   │   └── Periodic: *Object (3 properties)
+│   │   │       ├── MigrationState: *Object (3 properties)
+│   │   │       │   ├── StartTime: *string
+│   │   │       │   ├── Status: *Enum (4 values)
+│   │   │       │   │   ├── "Completed"
+│   │   │       │   │   ├── "Failed"
+│   │   │       │   │   ├── "InProgress"
+│   │   │       │   │   └── "Invalid"
+│   │   │       │   └── TargetType: *Enum (2 values)
+│   │   │       │       ├── "Continuous"
+│   │   │       │       └── "Periodic"
+│   │   │       ├── PeriodicModeProperties: *Object (3 properties)
+│   │   │       │   ├── BackupIntervalInMinutes: Validated<*int> (1 rule)
+│   │   │       │   │   └── Rule 0: Minimum: 0
+│   │   │       │   ├── BackupRetentionIntervalInHours: Validated<*int> (1 rule)
+│   │   │       │   │   └── Rule 0: Minimum: 0
+│   │   │       │   └── BackupStorageRedundancy: *Enum (3 values)
+│   │   │       │       ├── "Geo"
+│   │   │       │       ├── "Local"
+│   │   │       │       └── "Zone"
+│   │   │       └── Type: *Enum (1 value)
+│   │   │           └── "Periodic"
+│   │   ├── Capabilities: Object (1 property)[]
+│   │   │   └── Name: *string
+│   │   ├── Capacity: *Object (1 property)
+│   │   │   └── TotalThroughputLimit: Validated<*int> (1 rule)
+│   │   │       └── Rule 0: Minimum: -1
+│   │   ├── ConnectorOffer: *Enum (1 value)
+│   │   │   └── "Small"
+│   │   ├── ConsistencyPolicy: *Object (3 properties)
+│   │   │   ├── DefaultConsistencyLevel: *Enum (5 values)
+│   │   │   │   ├── "BoundedStaleness"
+│   │   │   │   ├── "ConsistentPrefix"
+│   │   │   │   ├── "Eventual"
+│   │   │   │   ├── "Session"
+│   │   │   │   └── "Strong"
+│   │   │   ├── MaxIntervalInSeconds: Validated<*int> (2 rules)
+│   │   │   │   ├── Rule 0: Maximum: 86400
+│   │   │   │   └── Rule 1: Minimum: 5
+│   │   │   └── MaxStalenessPrefix: Validated<*int> (2 rules)
+│   │   │       ├── Rule 0: Maximum: 2147483647
+│   │   │       └── Rule 1: Minimum: 1
+│   │   ├── Cors: Object (5 properties)[]
+│   │   │   ├── AllowedHeaders: *string
+│   │   │   ├── AllowedMethods: *string
+│   │   │   ├── AllowedOrigins: *string
+│   │   │   ├── ExposedHeaders: *string
+│   │   │   └── MaxAgeInSeconds: Validated<*int> (2 rules)
+│   │   │       ├── Rule 0: Maximum: 2147483647
+│   │   │       └── Rule 1: Minimum: 1
+│   │   ├── CreateMode: *Enum (2 values)
+│   │   │   ├── "Default"
+│   │   │   └── "Restore"
+│   │   ├── CustomerManagedKeyStatus: *string
```

Comment:
minor: Wondering if this should be blocked from the spec because it's actually get-only?